rotkehlchen/tests/unit/test_assets.py

```diff
@@ -633,3 +638,25 @@ def test_symbol_or_name(database):
     assert Asset('xyz').symbol_or_name() == 'custom name'
     with pytest.raises(UnknownAsset):
         Asset('i-dont-exist').symbol_or_name()
+
+
+def test_load_from_packaged_db(globaldb: GlobalDBHandler):
+    """Test that connecting to the packaged globaldb doesn't try to write into it."""
+    packaged_db_path = Path(__file__).resolve().parent.parent.parent / 'data' / 'global.db'
+    with TemporaryDirectory() as tmpdirname:
+        # Create a copy of the global db in a temp file
+        dest_file = Path(tmpdirname) / 'data' / 'global.db'
+        os.makedirs(dest_file.parent, exist_ok=True)
+        backup = shutil.copy(packaged_db_path, dest_file)
+
+        # mock Path parent attribute to return the destination file always
+        def parent():
+            return Path(tmpdirname)
+
+        # set the permissions for the copy of the globaldb to read only
+        backup.chmod(0o444)
+        # mock the parent method from pathlib in the initialization of the globaldb
+        with patch('pathlib.Path.parent', new_callable=PropertyMock) as mock_path:
```

Comment:
Can you also somehow make sure that the `dest_file` packaged_db is accessed in this test? Otherwise I gotta trust that the code you wrote is correct.

Reply:
Added the comment about the read permissions and added a read to ensure we can read from the packaged globaldb (is well connected)

Reply:
Still that's not what I meant.

What you are doing now is you are testing that you can read from the packaged global DB which does not tell you anything. I mean ofc you can. If you could not this entire thing would not work.

What I told you before and in discord is to: **Determine that you are reading from the `dest_file` global DB**. Essentially this will make sure your mock works. Otherwise you are just reading the normal packaged DB and the test has no effect.

The way I can imagine this being done is:
1. Add a special setting or something, or a custom price or anything directly to dest_file using sqlite.
2. At the end when you read from the packaged DB, make sure that thing you added is there.

This will achieve what you need. Which is to determine that the mock works fine.

Don't forget to explain with comments what you are doing.